static/app/components/dropdownMenu/item.tsx

```diff
@@ -201,7 +201,13 @@ function DropdownMenuItem({
       };
     }
 
-    return {as: 'div' as const};
+    return {
+      as: 'div' as const,
+      onClick: (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
fixes an issue where onAction events were propagating to parent anchor elements. Must have been doing this for us before